src/test/java/nl/tudelft/scrumbledore/BubbleTest.java

```diff
@@ -0,0 +1,41 @@
+package nl.tudelft.scrumbledore;
+
+//import static org.junit.Assert.*;
```

Comment:
Commented out assert imported are not necessary. Eclipse can (semi-)automatically import the correct assert test classes after you have added an assert.